core/src/apps/bitshares/base/helpers.py

```diff
@@ -0,0 +1,75 @@
+from trezor import wire
+from trezor.crypto import base58
+
+from apps.common import HARDENED
+
+
+def base58_encode(prefix: str, data: bytes) -> str:
+    return "{}{}".format(prefix, base58.encode(data))
+
+
+def base58_decode(prefix: str, data: str) -> bytes:
+    data = data.replace(prefix, "", 1)
+    return base58.decode(data)
+
+
+def unicodify(data: str):
```

Comment:
Do we need this?
I can recall I coded those solely for the purpose of STEEM messages

Reply:
Normally the only `string` I currently know is the `account_name`, you also used that, but probably in bitshares there are no unicode chars allowed?